README.md

```
@@ -34,13 +34,31 @@ OneScript позволяет создавать и выполнять текст
+
+Шаги установки:
 - Скачать ZIP-архив для Linux со [страницы релизов](https://github.com/EvilBeaver/OneScript/releases) или с [официального сайта](https://oscript.io).
 - Распаковать архив в удобный каталог.
 - Установить права на выполнение:
   ```bash
   chmod +x oscript
   ```
 
+Для FDD-варианта необходим .NET Runtime 8.0 — инструкция по установке на вашем дистрибутиве: [learn.microsoft.com/dotnet/core/install/linux](https://learn.microsoft.com/dotnet/core/install/linux).
+
+#### v1.x LTS — deb-пакет (Mono)
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx_closed; Actual: atx

(MD003, heading-style)